packages/theme/pages/Category.vue

```diff
@@ -247,7 +247,7 @@ export default {
       if (menu.value.isDisabled) {
         return '/c/' + category.slug;
       }
-      return category.link;
+      return category.link || '/c/' + category.slug;
```

Comment:
It's recommended to use string templates for increased readability.

```suggestion
      return category.link || `/c/${category.slug}`;
```
The above could be simplified to:
```js
return menu.value.isDisabled || !category.link
  ? `/c/${category.slug}`
  : category.link;
```

If you don't prefer a ternary operator, I'd suggest moving the category-based-on-slug route to its own variable to avoid repetition.
so:
```js
const slugBasedRoute = `/c/${category.slug}`;
if (menu.value.isDisabled) {
  return slugBasedRoute;
}
return category.link || slugBasedRoute;
```